pkg/scraper/scraper.go

```diff
@@ -107,12 +113,12 @@ type NodeInfo struct {
 }
 
 func (c *scraper) Scrape(baseCtx context.Context) *storage.MetricsBatch {
-	nodes, err := c.nodeLister.List(labels.Everything())
+	nodes, err := c.nodeLister.List(c.labelSelector)
 	if err != nil {
 		// report the error and continue on in case of partial results
 		klog.ErrorS(err, "Failed to list nodes")
 	}
-	klog.V(1).InfoS("Scraping metrics from nodes", "nodeCount", len(nodes))
+	klog.V(1).InfoS("Scraping metrics from nodes", "nodes", klog.KObjSlice(nodes), "nodeCount", len(nodes))
```

Comment:
Might be worth also logging the selector

Reply:
done